scripts/bash-scripts/vm.sh

```diff
@@ -0,0 +1,208 @@
+
+virt-install --name node-2 --memory 4096 --vcpus 2 --disk /var/lib/libvirt/images/node2.qcow2 --network bridge=m3-prov,model=virtio --osinfo detect=on --console pty,target_type=virtio --noautoconsole --boot nvram.template=/usr/share/OVMF/OVMF_VARS.fd --boot loader=/usr/share/OVMF/OVMF_CODE.secboot.fd --boot loader.secure=no --boot loader.type=pflash --boot loader.readonly=yes --debug -v --machine pc-q35-5.1
+
+virt-install --name node-3 --memory 4096 --vcpus 2 --disk /var/lib/libvirt/images/node3.qcow2 --network bridge=m3-prov,model=virtio --osinfo detect=on --console pty,target_type=virtio --noautoconsole --boot nvram.template=/usr/share/OVMF/OVMF_VARS.fd --boot loader=/usr/share/OVMF/OVMF_CODE.secboot.fd --boot loader.secure=no --boot loader.type=pflash --boot loader.readonly=yes --debug -v --machine pc-q35-5.1
+
+
+echo "Finished creating 3 virtual nodes"
+echo "Starting sushy-tools podman"
+
+cd $VBMCDIR
+
+
+cat << EOF > $VBMCDIR/sushy.config
+SUSHY_EMULATOR_AUTH_FILE = '$VBMCDIR/auth.conf'
+SUSHY_EMULATOR_SSL_CERT = '$VBMCDIR/cert.pem'
+SUSHY_EMULATOR_SSL_KEY = '$VBMCDIR/key.pem'
+SUSHY_EMULATOR_LISTEN_IP = '0.0.0.0'
+SUSHY_EMULATOR_VMEDIA_DEVICES = {
+    "Cd": {
+        "Name": "Virtual CD",
+        "MediaTypes": [
+            "CD",
+            "DVD"
+        ]
+    },
+    "Floppy": {
+        "Name": "Virtual Removable Media",
+        "MediaTypes": [
+            "Floppy",
+            "USBStick"
+        ]
+    }
+}
+EOF
+
+# Need this line added to /etc/hosts for sushy-tools to work properly
+LINE="192.168.124.99 boot.ironic.suse.baremetal api.ironic.suse.baremetal inspector.ironic.suse.baremetal media.suse.baremetal"
+FILE="/etc/hosts"
+grep -qF "$LINE" "$FILE" || echo "$LINE" >> "$FILE"
+
+
+htpasswd -b -B -c auth.conf foo foo
+
+openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -sha256 -days 365 --noenc -subj "/C=/ST=/L=/O=/OU=/CN="
+
+
+# We run sushy-tools in podman so that it is running in the background
+
+sudo podman run -d --rm --privileged  --name sushy-tools   -v ${HOME}/metal3-demo/vbmc:$VBMCDIR:Z   -v /var/run/libvirt:/var/run/libvirt:Z   -e SUSHY_EMULATOR_CONFIG=$VBMCDIR/sushy.config   -p 8000:8000   quay.io/metal3-io/sushy-tools:latest sushy-emulator
+echo "Finished starting sushy-tools podman"
+
+echo "Sleeping for 10 seconds to make sure podman has started"
+sleep 10s
+
+# We automatically grab the mac address of each vm and the sushy-tools id of each vm
+
+NODE1ID=$(curl -L https://$IP_ADDR:8000/redfish/v1/Systems/node-1 -k -u "foo:foo" | jq -r '.UUID')
+NODE2ID=$(curl -L https://$IP_ADDR:8000/redfish/v1/Systems/node-2 -k -u "foo:foo" | jq -r '.UUID')
+NODE3ID=$(curl -L https://$IP_ADDR:8000/redfish/v1/Systems/node-3 -k -u "foo:foo" | jq -r '.UUID')
+
+echo $NODE1ID
```

Comment:
Adding a meaningful message e.g. `Node 1 ID: $NODE1ID` can be helpful when debugging. Otherwise, we'd be seeing empty lines if the value is not properly populated. Applies to the MAC addresses below as well.